### PR TITLE
doc/user: added fill property to svgs

### PR DESCRIPTION
**Pull request by joacoc**
### Motivation

Nothing more than this lil two details in the documentation:

- Added `fill` property to the svgs in `Tools and Integrations` section.
- And centered the text under the SVG for cases where the text is long and it breaks.

### Testing

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.

### Release notes

This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):

  -

### CODE REVIEW

File: doc/user/layouts/shortcodes/supportLevel.html [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<svg class="level-of-support level-{{ .Get 0 }}" xmlns="http://www.w3.org/2000/svg">
-  <use href="#icon-{{ .Get 0 }}" />
-</svg>
-{{ replace (.Get 0) "-" " " | title }}
+<div style="width: 100%; height: 100%; text-align: center;">
+  <svg class="level-of-support level-{{ .Get 0 }}" xmlns="http://www.w3.org/2000/svg">
+    <use href="#icon-{{ .Get 0 }}" />
+  </svg>
+  <p>{{ replace (.Get 0) "-" " " | title }}</p>
```
Comment (ruf-io) [inline doc/user/layouts/shortcodes/supportLevel.html:5]: when you wrap the text in a <p> tag the font size goes from 14px (applied to <td>) to 15px (applied to <p>) intentional?

**Comment by ruf-io** [comment]
Was this motivated by the icons looking aliased or wrong in certain browsers? I almost want to get rid of the icons entirely and just use text.  Due to weird SVG rendering behavior, adding the fill seems to actually make the icons (especially the beta ones) super faint:

Before:
image

After: 
image

**Comment by joacoc** [comment]
Yeah. They were perfect on the laptop but different on a second screen. I was trying to fix them, but the Beta and Alpha now look brighter than the sun.

> I almost want to get rid of the icons entirely and just use text

This got me thinking another alternatives like lines ( text for the color blindless and colors but more minimal ):

Screen Shot 2022-03-24 at 10 33 07
.

**Comment by morsapaes** [comment]
Just noticed this PR hanging. 

I'd be +1 to replacing the icons with something more subdued. I like the lines, or even just a filled circle! A while back, I ran into [these badges](https://gitlab.com/KazukyAkayashi/hugo-badges-shortcodes#result) and it could also be an option (+ to up our emoji game! 👹), though it might make the page equally as heavy.

**Comment by ruf-io** [comment]
I am open to anything that meets the following requirements:

1. Gives a quick scannable status indicator to the user
2. Works for colorblind users who can't differentiate between green and red.

I think the badges fit that description, or emoji's. But not solid color circles.

**Comment by joacoc** [comment]
Closing. Outdated PR.

**Pull request closed without merging**